client/core/core.go

```diff
@@ -4396,17 +4413,19 @@ func (c *Core) MaxBuy(host string, base, quote uint32, rate uint64) (*MaxOrderEs
 	maxBuy, err := quoteWallet.MaxOrder(&asset.MaxOrderForm{
 		LotSize:       quoteLotEst,
 		FeeSuggestion: swapFeeSuggestion,
-		AssetConfig:   quoteAsset,
-		RedeemConfig:  baseAsset,
+		AssetVersion:  quoteAsset.Version, // using the server's asset version, when our wallets support multiple vers
```

Comment:
Eventually, we'll get the UI figured out where we hide order forms for unsupported versions, and will also check version compatibility before we get this far. This note is good enough for now.

Reply:
Yeah it's not yet ideal to just get an error from the API, but at least it's caught here and in `PreOrder` when it calls `walletSet` and returns for `!versCompat`.